packages/ui/components/Avatars/index.tsx

```diff
@@ -12,21 +13,20 @@ export const PersonAvatar: React.FC<PersonAvatarProps> = ({ personId }) => {
 };
 
 interface ProfileAvatar {
-  userId: string;
   imageUrl?: string | null;
+  email: string;
 }
 
-export const ProfileAvatar: React.FC<ProfileAvatar> = ({ userId, imageUrl }) => {
-  if (imageUrl) {
-    return (
-      <Image
-        src={imageUrl}
-        width="40"
-        height="40"
-        className="h-10 w-10 rounded-full object-cover"
-        alt="Avatar placeholder"
-      />
-    );
-  }
-  return <Avatar size={40} name={userId} variant="bauhaus" colors={colors} />;
+export const ProfileAvatar: React.FC<ProfileAvatar> = ({ imageUrl, email }) => {
+  const imageSrc = imageUrl || `https://www.gravatar.com/avatar/${sha256(email)}?d=retro`;
+
+  return (
+    <Image
```

Comment:
Since you are using the next.js image component you need to whitelist the gravatar domain in the `apps/web/next.config.mjs` config :-) 
(`nextConfig.images.remotePatterns`)

Reply:
✅